utils/notification_service.py

```diff
@@ -547,7 +547,8 @@ def payload(self) -> str:
                     items = re.findall(pattern, line)
                 elif "tests/models/" in line:
                     model = line.split("/")[2]
-                    new_failed_tests[model] = {"single-gpu": [], "multi-gpu": []}
+                    if model not in new_failed_tests:
+                        new_failed_tests[model] = {"single-gpu": [], "multi-gpu": []}
```

Comment:
This is a fix for **not having all new failing tests being saved into** `new_model_failures.json`